frontend/src/app/services/history-query.service.ts

```diff
@@ -219,6 +219,16 @@ export class HistoryQueryService {
     );
   }
 
+  /**
+   * Ckeck correspondence between the delimiter entered and the delimiter in the CSV file
+   */
+  checkCorrespondence(file: File, delimiter: string) {
+    const formData = new FormData();
+    formData.set('file', file);
+    formData.set('delimiter', delimiter);
+    return this.http.post<{ correspond: boolean }>(`/api/history-queries/items/check-correspondence`, formData);
+  }
```

Comment:
I'm not sure we need this : for export, we just use a delimiter from a list, and for the import, we just throw an error when checking the file, and we display the error on the frontend.

Reply:
Same remark for south-connector